client/src/router/index.js

```diff
@@ -6,10 +6,11 @@ import Register from '@/components/Register';
 import Topics from '@/components/Topics';
 import Profile from '@/components/Profile';
 import Test from '@/components/Test';
+import store from '@/store/store';
 
 Vue.use(Router);
 
-export default new Router({
+const router = new Router({
   routes: [
     {
```

Comment:
You can style this to reduce vertical space usage by moving parenthesize 1 line up
 
```js
router: [{
  path: 'something'
}, { 
path: 'something'
}